CHANGELOG.md

```diff
@@ -17,6 +17,25 @@ For reference, the possible headings are:
 
 ## [unreleased]
 
+## [0.10.0] - 2020-09-22
+
+### Added
+
+- Redis support: `xaynet_server` now stores PET data in redis (https://github.com/xaynetwork/xaynet/pull/416, https://github.com/xaynetwork/xaynet/pull/515)
```

Comment:
I havn't merge the integration yet because @finiteprods wanted to have a look but we could write:
Preparation for Redis support

Reply:
done